src/utils/changes.ts

```diff
@@ -175,3 +181,152 @@ export function prependChangeset(
 
   return markdown.slice(0, startIdx) + newSection + markdown.slice(startIdx);
 }
+
+interface Commit {
+  hash: string;
+  title: string;
+  hasPRinTitle: boolean;
+  pr: string | null;
+  milestone: string | null;
+}
+
+interface Milestone {
+  title?: string;
+  description?: string;
+  state?: string;
```

Comment:
What are the possible `state` values? Are they unlimited? Can we type this, or link to the place where these are documented?

Reply:
I'll add typing. They can only be `OPEN` or `CLOSED`.